docs/fides/docs/api/index.md

```diff
@@ -14,18 +14,20 @@ The `fidesctl` API is almost completely programmatic, so it's easier to grasp th
 
 * Except for the `DELETE`, the endpoints accept and/or return JSON objects that represent the named resource. The structure of these objects is given in the [Fides Language: Resources chapter](/language/resources/organization/) -- it's the same structure that's used in the resource manifest files.
 
-That's about all there is to it. There are an additional four endpoints that we'll look at below, but the sets of quintuplet endpoints listed above make up the core of the `fidesctl` API.
+That's about all there is to it. There are an additional six endpoints that we'll look at below, but the sets of quintuplet endpoints listed above make up the core of the `fidesctl` API.
 
-After a brief review of the four addition endpoints, we'll provide a complete API reference followed by a set of cURL calls that you can use to exercise the API on your system.
+After a brief review of the six additional endpoints, we'll provide a complete API reference followed by a set of cURL calls that you can use to exercise the API on your system.
```

Comment:
god help us for manually updating this lol, i don't know of a better way though unless we just make the auto-generated API docs more robust, which is probably the right answer? for another time though

Reply:
Honestly we could scrap everything on this page except the Swagger-generated reference docs. I don't think we need to hand-hold users through understanding the API "formula". A well-designed API contract should speak for itself 🤷‍♂️

Reply:
@iamkelllly feelings on this? I'm in agreement here as well, the FidesAPI is really only meant to be touched via the CLI, not extensible or a proper service like fidesops. These docs are going to get out of sync quickly

Reply:
regardless we'll revisit in another PR